reporter/prometheus_reporter_unix.mli

```diff
@@ -0,0 +1,55 @@
+(** Report metrics for Prometheus.
+    See: https://prometheus.io/
+
+    Notes:
+
+    - This module is intended to be used by applications that export Prometheus metrics.
+      Libraries should only link against the `Prometheus` module.
+
+    - This module automatically initialises itself and registers some standard collectors relating to
+      GC statistics, as recommended by Prometheus.
+
+    - This extends [Prometheus_reporter] with support for cmdliner option parsing, a server pre-configured
+      for Unix, and a start-time metric that uses [Unix.gettimeofday].
+ *)
+
+type config = int option
+
+val opts : config Cmdliner.Term.t
+(** [opts] is the extra command-line options to offer Prometheus
+    monitoring. *)
```

Comment:
It might make more sense to keep the cmdliner stuff for cohttp in app. That would also avoid exposing the type (which is likely to change - see #51).